### PR TITLE
A bunch of small fixes

### PR DESCRIPTION
**Pull request by feross**
See individual commits

### CODE REVIEW

**Comment by feross** [comment]
Review: @dcposch

File: src/renderer/lib/migrations.js [hunk 1]
```diff
@@ -95,15 +91,15 @@ function migrate_0_7_0 (saved) {
 }
 
 function migrate_0_7_2 (saved) {
-  if (!saved.prefs) {
```
Comment (dcposch) [inline src/renderer/lib/migrations.js]: Was this a bug before? Seems like both ways should work.

Usually I wouldn't change migration steps that have already run

**Comment by dcposch** [comment]
LGTM

File: src/renderer/lib/migrations.js (hunk 1 above)
Comment (feross) [inline src/renderer/lib/migrations.js]: Yeah, don't want to get in a habit of changing these after-the-fact, but it looked weird that every migration had a different way of checking for undefined (`!condition` vs. `=== undefined` vs. `== null`).

What we really want is `== null`, IMO.

Now that it's consistent, it's clear what to use in future migrations.

**Pull request merged**